dataland-frontend/src/components/resources/frameworkDataSearch/DisplayFrameworkDataTable.vue

```diff
@@ -9,12 +9,14 @@
       :sortOrder="1"
       sortMode="single"
       :expandableRowGroups="true"
+      :headerInputStyle="headerInputStyle"
       v-model:expandedRowGroups="expandedRowGroups"
     >
       <Column
         bodyClass="headers-bg"
-        headerStyle="width: 30vw;"
+        bodyStyle="width: 30vw;"
```

Comment:
Wurde `width: 30vw` zu bodyStyle geswitcht, damit man in den headerStyle einfach was reinpluggen kann?
Funktional sollte das ja nichts ändern, richtig?

Reply:
Ja genau. Und ich glaube wir schmeißen momentan auch für lksg width: 30vw als Input in die Table Component.